internal/pkg/template/templates/addons/aurora/env/addons.parameters.yml

```diff
@@ -1,9 +1,3 @@
 Parameters:
-  VPCID: 
-    - !If IsVPCImported
-    - !Ref ImportedVPCID
-    - !Ref VPC
-  PrivateSubnets:
-    - !If IsVPCImported
-    - !Ref ImportedPrivateSubnets
-    - !Join [ ',', [ !Ref PrivateSubnet1, !Ref PrivateSubnet2 ] ]
+  VPCID: !Ref VPC
```

Comment:
dumb question, do we need to update the CF templates as well in `aurora/env/serverlessv2.yml` with these parameters?

Reply:
No need! Those templates just make use of the `VPCID` parameter like this
 https://github.com/aws/copilot-cli/blob/b69393aa9e504ac3b398c53adef360e3f1ab0fb0/internal/pkg/template/templates/addons/aurora/env/serverlessv2.yml#L51